docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.9'
+services:
+    stargate:
+        image: 'stargateio/stargate-3_11:v1.0.0'
```

Comment:
In the [docs](https://stargate.io/docs/stargate/1.0/quickstart/quick_start-document.html) they use `stargateio/stargate-3_11:v1.0.13`
Do you think this matters?

Reply:
ah good spot! 👍 looks like they have bumped the build versions, probably more important when they move minor versions